packages/rpms/SPECS/wazuh-server.spec

```diff
@@ -205,6 +205,7 @@ rm -fr %{buildroot}
 %dir %attr(750, root, wazuh) %{_localstatedir}var/lib/wazuh-server/engine
 %dir %attr(750, root, wazuh) %{_localstatedir}var/lib/wazuh-server/engine/tzdb
 %dir %attr(750, root, wazuh) %{_localstatedir}var/log/wazuh-server
+%dir %attr(750, root, wazuh) %{_localstatedir}var/log/wazuh-server/engine
```

Comment:
Reason of that directory ?

Reply:
That folder needed to be added to make the engine output files work in this PR: https://github.com/wazuh/wazuh/pull/26363

That directory was created but not installed because that directive was missing.